tests/scripts/thread-cert/node_cli.py

```diff
@@ -441,13 +441,17 @@ def get_addrs(self):
         return addrs
 
     def get_addr(self, prefix):
-        network = ipaddress.ip_network(unicode(prefix))
+        network = ipaddress.ip_network(u'%s' % str(prefix))
         addrs = self.get_addrs()
 
         for addr in addrs:
-            ipv6_address = ipaddress.ip_address(addr.decode("utf-8"))
+            if isinstance(addr, bytearray):
+                addr = bytes(addr)
+            elif isinstance(addr, str) and sys.version_info[0] == 2:
+                addr = addr.decode("utf-8")
```

Comment:
single quote: `"utf-8"` -> `'utf-8'`

Reply:
All other `utf-8` instances in this file use double quotes. Will clean this up separately.